src/main/java/org/b612foundation/adam/opm/OdmCommonMetadata.java

```diff
@@ -66,8 +66,18 @@
    * set as needed.
    */
   public enum TimeSystem {
-    /** Universal coordinated time. */
-    UTC
+    /** Coordinated Universal Time. */
```

Comment:
Fix indentation.

Reply:
was this tabs vs spaces? Indentation looks fine... I just replaces with spaces though -